hipify-clang/src/CUDA2HipMap.cpp

```diff
@@ -397,12 +396,11 @@ const std::map <llvm::StringRef, hipCounter> CUDA_INCLUDE_MAP{
     // CUDNN includes
     {"cudnn.h",                   {"hipDNN.h",                 CONV_INCLUDE_CUDA_MAIN_H, API_DNN}},
 
-    // CUDNN includes
+    // CUFFT includes
     {"cufft.h",                   {"hipfft.h",                 CONV_INCLUDE_CUDA_MAIN_H, API_FFT}},
 
-    // HIP includes
-    // TODO: uncomment this when hip/cudacommon.h will be renamed to hip/hipcommon.h
```

Comment:
do we need this todo?

Reply:
No, we don't. It is appeared with SHOC hipification which now works fine with included cudacommon.h.